tests/buildtest/mod.rs

```diff
@@ -83,14 +83,18 @@ fn test_sandbox_oom() {
 }
 
 #[test]
-fn test_cargo_config() {
+fn test_override_files() {
     runner::run("cargo-config", |run| {
         run.build(SandboxBuilder::new().enable_networking(false), |build| {
             let storage = rustwide::logging::LogStorage::new(LevelFilter::Info);
             rustwide::logging::capture(&storage, || -> Result<_, Error> {
-                build.cargo().args(&["run"]).run()?;
+                build.cargo().args(&["--version"]).run()?;
```

Comment:
Can you run both `cargo run` and `cargo version`? You can do that in two different log captures if that makes the test easier to write. Otherwise here you're not testing the `.cargo/config` with the invalid target in it.

Reply:
It's tested below: https://github.com/rust-lang/rustwide/pull/60/files/ebb89f4c06a93999434e389b4e38b55b86aebaf7#diff-fb017238a25c2e8d919e32fe773c81046e9773a09a1be46b8bfa752f0cd6593aR97. It's just that the logging isn't captured (because it doesn't need to be).